Cargo.toml

```diff
@@ -461,5 +461,3 @@ transaction-fuzzer = { path = "crates/transaction-fuzzer" }
 typed-store = { path = "crates/typed-store" }
 typed-store-derive = { path = "crates/typed-store-derive" }
 typed-store-error = { path = "crates/typed-store-error" }
-
-[patch.crates-io]
```

Comment:
You are removing it because the patch is already adding it? If yes, it's actually allowed to have it twice?

Reply:
Yes, it is just not needed